docs/deploying_server.md

```diff
@@ -0,0 +1,108 @@
+# Deploying Model Server {#ovms_docs_deploying_server}
+
+1. Docker is the recommended way to deploy OpenVINO Model Server. Pre-built container images are available on Docker Hub and Red Hat Ecosystem Catalog. 
+2. Host Model Server on baremetal.
+3. Deploy OpenVINO Model Server in Kubernetes via helm chart, Kubernetes Operator or OpenShift Operator.
+
+## Deploying Model Server in Docker Container 
+
+This is a step-by-step guide on how to deploy OpenVINO&trade; Model Server on Linux, using a pre-build Docker Container. 
+
+**Before you start, make sure you have:**
+
+- [Docker Engine](https://docs.docker.com/engine/) installed 
+- Intel® Core™ processor (6-12th gen.) or Intel® Xeon® processor
+- Linux, macOS or Windows via [WSL](https://docs.microsoft.com/en-us/windows/wsl/) 
+- (optional) AI accelerators [supported by OpenVINO](https://docs.openvino.ai/2022.2/openvino_docs_IE_DG_supported_plugins_Supported_Devices.html). Accelerators are tested only on bare-metal Linux hosts.
+
+### Launch Model Server Container 
+
+This example shows how to launch the model server with a ResNet50 image classification model from a cloud storage:
+
+#### Step 1. Pull Model Server Image
+
+Pull an image from Docker: 
+
+```bash
+docker pull openvino/model_server:latest
+```
+
+or [RedHat Ecosystem Catalog](https://catalog.redhat.com/software/containers/intel/openvino-model-server/607833052937385fc98515de):
+
+```
+docker pull registry.connect.redhat.com/intel/openvino-model-server:latest
+```
+
+#### Step 2. Prepare Data for Serving
+
+```bash
+# start the container with the model
+docker run -p 9000:9000 openvino/model_server:latest \ 
+--model_name resnet --model_path gs://ovms-public-eu/resnet50-binary \ 
+--layout NHWC:NCHW --port 9000 
+
+# download input files: an image and a label mapping file
+wget https://raw.githubusercontent.com/openvinotoolkit/model_server/releases/2022/1/demos/common/static/images/zebra.jpeg
+wget https://raw.githubusercontent.com/openvinotoolkit/model_server/releases/2022/1/demos/common/python/classes.py
+
+# install the Python-based ovmsclient package
+pip3 install ovmsclient
+```
+
+#### Step 3. Run Prediction
+
+
+```bash
+echo 'import numpy as np
+from classes import imagenet_classes
+from ovmsclient import make_grpc_client
+
+client = make_grpc_client("localhost:9000")
+
+with open("zebra.jpeg", "rb") as f:
+   img = f.read()
+
+output = client.predict({"0": img}, "resnet")
+result_index = np.argmax(output[0])
+print(imagenet_classes[result_index])' >> predict.py
+
+python predict.py
```

Comment:
please add expected output  in the next line (zebra) so the output can be checked with tests

Reply:
thanks, added